### PR TITLE
PWN-5022- Solend withdraw UI

### PR DESCRIPTION
**Pull request by 1sakov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-5022

## Description of Work

[ what work was done in this ticket ]

## Screenshots (Optional)

Screenshot #1 | Screenshot #2 | Screenshot #3
:------------:|:-------------:|:------------:
[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]

### CODE REVIEW

**Review by gslevinkov — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/solend/ui/deposits/SolendUserDepositsFragment.kt [hunk 1]
```diff
@@ -61,4 +65,11 @@ class SolendUserDepositsFragment :
             tokensRecyclerView.isVisible = !isLoading
         }
     }
+
+    override fun onAddMoreClicked(token: SolendDepositToken.Active) {
```
Comment (davran1001) [inline app/src/main/java/org/p2p/wallet/solend/ui/deposits/SolendUserDepositsFragment.kt:69]: is there any implementation, if not maybe left TODO here

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
+#Tue Oct 11 15:00:33 GMT+06:00 2022
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```
Comment (davran1001) [inline gradle/wrapper/gradle-wrapper.properties:3]: is GRADLE_USER_HOME local env ? if yes, i think we need to rollback this

**Review by davran1001 — APPROVED** [review]

**Review by kabramovp2p — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/solend/ui/deposits/SolendUserDepositsFragment.kt (hunk 1 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/solend/ui/deposits/SolendUserDepositsFragment.kt:69]: no, the action is here - https://github.com/p2p-org/p2p-wallet-android/pull/849
this needs to be merged asap

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (1sakov) [reply gradle/wrapper/gradle-wrapper.properties:3]: no, not local.
this is fix for everyone with Dolphin AS

**Pull request merged**